packages/web-components/fast-components/src/search/fixtures/search.html

```diff
@@ -40,7 +40,13 @@ <h2>Minlength</h2>
 <!-- Start -->
 <h2>With start</h2>
 <fast-search>
-    <svg slot="start" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16">
+    <svg
+        slot="start"
+        xmlns="http://www.w3.org/2000/svg"
+        width="16"
```

Comment:
I realize this is the test fixture, but for standard guidance we don't include the width and height anymore.